intermediate/xarray_ecosystem.ipynb

```diff
@@ -20,16 +33,141 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## The accessor interface\n",
+    "## (Preface) The accessor interface\n",
```

Comment:
I would move this to the end as it's a little advanced for our main audience at the moment. It can be optional material.

Reply:
I wanted to mention the accessors at the start so people knew where the `da.pint.quantify` etc. syntax came from, so I've split the accessor content into a quick note at the top and a more detailed explanation for developers at the bottom.